samples/MerQure.Tools.Samples/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MerQure.Tools.Samples.RetryBusExample.Domain;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace MerQure.Tools.Samples
+{
+	public class Program
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1118)